.github/workflows/auto-merge.yml

```
@@ -0,0 +1,14 @@
+name: auto-merge
+
+on:
+  pull_request:
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2
+        with:
+          target: minor
```

I'd like to make the "approve by default" option explicit.
```suggestion
          target: minor
          approve: true
```

---

I'm in favor of making that more explicit